src/components/DateInput.jsx

```diff
@@ -81,13 +87,29 @@ export default class DateInput extends React.Component {
   }
 
   onKeyDown(e) {
```

Comment:
`lodash.debounce` is just that one function, not an "entire library" - imo it's always better to add a dep rather than reimplementing something yourself.